### ISSUE TITLE
Understand conditional versioning

### ISSUE THREAD

**Issue by intracer** [fthomas/scala-steward#809]
Example:
https://github.com/intracer/scalawiki/pull/96/commits/5e5e398bfa5f6dfc8479c016c4ab18735ea6827f
Scala Steward changed  code that conditionally used either "2.7.4" or "2.6.13": 
      `def PlayJsonV(isScala213: Boolean) = if (isScala213) "2.7.4" else "2.6.13"` 
to code that in both branches uses the same latest version:
      `def PlayJsonV(isScala213: Boolean) = if (isScala213) "2.7.4" else "2.7.4"`

**Comment by exoego** [comment]
I think you can allow Steward to update `2.7.4` while keeping `2.6.13` using `scala-steward:off` markup.

```scala
val playJsonOld = "2.6.13" // scala-steward:off
def PlayJsonV(isScala213: Boolean) = if (isScala213) "2.7.4" else playJsonOld 
```

Ref. https://github.com/fthomas/scala-steward/blob/651fd2c6b0a9ada834850ab251247405c6a09b30/docs/repo-specific-configuration.md#ignore-lines

**Comment by intracer** [comment]
@exoego thank you for the advice. 
However if there comes new release in 2.6 series, one may want to update it too (2.6.13 to 2.6.14 and 2.7.4 to 2.7.5 independently)

**Comment by fthomas** [comment]
My feeling is that supporting this would require quite some changes in Scala Steward. Instead of getting dependencies and updates for `scalaVersion`, it would need to get dependencies and updates for all `crossScalaVersions` and remember which dependencies use different versions for different Scala versions.

**Comment by fthomas** [comment]
FYI: I think https://github.com/fthomas/scala-steward/pull/1174 is an important milestone towards supporting this feature.

**Comment by fthomas** [comment]
It seems that #1209 fixes this issue. I'm using that branch for my public Scala Steward instance and it created these two PRs:
 * https://github.com/pureconfig/pureconfig/pull/677/files
 * https://github.com/pureconfig/pureconfig/pull/678/files

Both changed this line:
```
"io.circe" %% "circe-core" % forScalaVersions { case (2, 11) => "0.11.1"; case _ => "0.12.3" }.value,
```
The first PR updated the Circe version for Scala 2.11 and the second updated the Circe version for Scala >= 2.12.

**Comment by fthomas** [comment]
Fixed by #1209.
